hack/unwanted-dependencies.json

```diff
@@ -252,22 +232,15 @@
       "golang.org/x/exp": [
         "github.com/antlr4-go/antlr/v4",
         "github.com/google/cel-go",
+        "github.com/grpc-ecosystem/go-grpc-middleware/v2",
```

Comment:
this is new and unexpected, need to chase this and eliminate it

Reply:
sorry, I should have been clearer, etcd's switch to `go-grpc-middleware/v2` was completely expected and desired

`go-grpc-middleware/v2` adding a dependency onto `golang.org/x/exp` is new and unexpected, and I need to go chase that in *that* dep